File: BuySell/BuySell.js
function BuySell(prices){
    let min = prices[0] , max = 0;
    for(i=0; i<= prices.length; i++){
        if(prices[i] - min > max){
            max = prices[i] - min;
        }
        if(prices[i]< min){
            min = prices[i]
        }
    }
    console.log(max);
}

BuySell([7,1,5,3,6,4])

File: BuySell/BuySell.py
def BuySell(prices):
    min = prices[0]
    max = 0
    for i in range(len(prices)):
        if min - prices[i] > max:
            max = min - prices[0]
        if min - prices[i] > max:
             max = prices[i]
    print(max)

BuySell([7,1,5,3,6,4])